### PR TITLE
Issue#5

### PR DESCRIPTION
**Pull request by dmitriim**
Internal review for https://github.com/SREd-URV/moodle-auth_ip/issues/5

### CODE REVIEW

**Comment by brendanheywood** [comment]
General issues:
- [ ] I plugged my ipv6 address right in and it didn't work:

image
- [ ] actually it flat out just doesn't work at all!

image
- [ ] remove the old unused lang pack items (check this out: https://wiki.wgtn.cat-it.co.nz/wiki/User:Brendan/Cheat_Sheet#XARGS_with_multiple_commands_piped_inside_xargs )

File: auth.php [hunk 1]
```diff
@@ -51,115 +51,221 @@ function __construct() {
...
      *
-     * @see https://secure.php.net/manual/en/ref.network.php#74656
-     * @param $ip
      * @return bool
      */
-    function is_ip_valid($ip) {
-        // List of allowed IP addresses or CIDR ranges
-        $valid_ips_or_cidrs = explode(',', str_replace(' ', '', $this->config->valid_ips));
+    public function is_internal() {
+        return false;
```
Comment (brendanheywood) [inline auth.php:77]: Why did this change? it was prevously true, and should be true

Comment (dmitriim) [inline auth.php:77]: Look at 122 line. It was false. Haven't changed this at all.

File: cli/check_before_logging.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    cli_error(get_string('cliunknowoption', 'admin', $unrecognized));
+}
+
+
+if ($options['help'] || (empty($options['enable']) && empty($options['disable']))) {
+    $help = "
+Command line script to enable/disable \"Check IP before logging in\" setting for auth/ip plugin.
+
+Options:
+--enable       No interactive questions or confirmations
```
Comment (brendanheywood) [inline cli/check_before_logging.php]: ??

File: cli/check_before_logging.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+--disable      Upgrade even if the version is not marked as stable yet,
```
Comment (brendanheywood) [inline cli/check_before_logging.php]: ??

File: cli/check_before_logging.php (hunk 2 above)
Comment (dmitriim) [inline cli/check_before_logging.php]: Good catch. Copy and paste issue. Will fix

**Pull request merged**